src/request/containers/RequestDetailPanelDataContainer.ts

```diff
@@ -0,0 +1,21 @@
+import { DataComponent, IDataComponentProps } from '../components/RequestDetailPanelData';
+import { IRequestState } from '../stores/IRequestState';
+import { getTotalOperationCount } from '../selectors/RequestDetailDataSelectors';
+
+import { connect } from 'react-redux';
+
+function mapStateToProps(state: IRequestState): IDataComponentProps {
+    return {
+        totalOperationCount: getTotalOperationCount(state)
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+    };
+}
+
+export = connect(
+    mapStateToProps,
+    mapDispatchToProps
+)(DataComponent);
```

Comment:
The [`react-redux` documentation](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options) states that, if `mapDispatchToProps` is omitted, `connect` will then map `dispatch` into the wrapped component's properties.  While that probably wouldn't hurt, it's not exactly what I would have expected, so I explicitly pass a "dummy" implementation.